CHANGELOG.md

```diff
@@ -5,8 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
-
+## [Unreleased]
+

```

Comment:
I'm not sure if these line breaks work (it probably works), as we usually put the line right next to each other. 

If possible, check later how the release process modified the CHANGELOG